netbox/extras/tables.py

```diff
@@ -204,6 +212,15 @@ class ObjectChangeTable(BaseTable):
         linkify=True,
         format=settings.SHORT_DATETIME_FORMAT
     )
+    user_name = tables.TemplateColumn(
+        verbose_name='Username',
+        orderable=False
+    )
```

Comment:
Not that much for ordering. I can't see the point in preventing ordering on this column, so I'll remove it.
I intended to name the column as "Username", because the column name would have defaulted to "User name" which, in regard to "Full name", was reading weird.
I'll make some changes, and let you decide whether to close the conversation or not.